### DISCUSSION TITLE
Valheim Dedicated Server

### DISCUSSION THREAD

**Discussion by Technoli0** [ebkr/r2modmanPlus#414] [Q&A]
Is there a way to use this with a valheim Dedicated Server? I tried manually mapping it to the C:\Program Files (x86)\Steam\steamapps\common\Valheim dedicated server directory but then it complained that the directory must have the valheim.exe file.

**Comment by ebkr** [comment]
Not currently although server support is something I'd like to add.

**Reply by Rumaan3012** [reply]
Were you ever able to add in dedicated server support for Valheim?
